generated/java/google-cloud-speech-v1beta1/src/main/java/com/google/cloud/speech/spi/v1beta1/SpeechApi.java

```diff
@@ -146,6 +154,11 @@ public void close() throws IOException {
     }
   }
 
+  public final StreamObserver<StreamingRecognizeResponse> streamingRecognize(
+      StreamObserver<StreamingRecognizeRequest> streamObserver) {
+    return streamingRecognizeCallable.call(streamObserver);
+  }
```

Comment:
Right now streamingRecognizeCallable is a SimpleApiCallable object. The logic is the same as the other simple API calls except the request/response types are StreamObserver<T>